configuration/pih/petl/jobs/datawarnings/aggregrate/job.yml

```diff
@@ -0,0 +1,16 @@
+type: "sqlserver-bulk-import"
+configuration:
+  extract:
+    datasource: "mysql/openmrs.yml"
+    query:  "datawarnings/aggregrate/source.sql"
+
+  load:
+    datasource: "sqlserver/openmrs_extractions.yml"
+    table: "hivmigration_aggregrate_data_warnings"
+    schema: "datawarnings/aggregrate/target.sql"
+
+  dropAndRecreateTable: false
+  conditional: "SELECT COUNT(*)=1 FROM information_schema.tables WHERE table_schema = 'openmrs' AND table_name = 'hivmigration_data_warnings'"
```

Comment:
Note that this depends on the database being names "openmrs"... which should be fine for all our production instances, but may cause issues for our local dev environments.  I was thinking we should support some sort of environ variable search and replace with the conditional (and all the scripts, for that matter), but didn't add it at this point.

Reply:
(Also the use case for this is for the other, non-HIV migration servers in the ZL network... we don't want this job to fail just because that table isn't present on those servers)